db/migrate/20170824203630_add_unique_code_to_offences.rb

```diff
@@ -1,7 +1,7 @@
 class AddUniqueCodeToOffences < ActiveRecord::Migration
   def up
     add_column :offences, :unique_code, :string, default: 'anyoldrubbish', null: false
-    Rake::Task['data:migrate:offence_unique_code'].invoke
+    Rake::Task['data:migrate:offence_unique_code_scheme_9'].invoke
```

Comment:
I've just noticed this is already in, but the [comment](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2357/files#r186106020) also applies here.